xcode/Subconscious/Shared/Components/Detail/MemoViewerDetailView.swift

```diff
@@ -261,6 +272,25 @@ extension MemoViewerDetailAction: CustomLogStringConvertible {
     }
 }
 
+/// React to actions from the root app store
+extension MemoViewerDetailAction {
+    static func fromAppAction(
+        action: AppAction,
+        description: MemoViewerDetailDescription
```

Comment:
Same comment as above. Best to avoid passing in state. Instead, include relevant state in action being mapped.